trillian/migrillian/README.md

```diff
@@ -0,0 +1,19 @@
+Migrillian Tool
+===============
+
+*Migrillian* is a tool that transfers data from Certificate Transparency logs to
```

Comment:
Maybe insert "other" or "existing" before Certificate Transparency so its clearer what it's for.

Reply:
Done.